Move /usage/distributed-tracing to /tracing (#9572)
getsentry/sentry-docs: platform-includes/tracing/custom-instrumentation/dotnet.mdx

> On this page you will learn how to manually propagate trace information into and out of your .NET application. Please note, that you do not need to do this manually, if you [use one of our supported frameworks](/platforms/dotnet/tracing/#how-to-use-tracing), or you have our <PlatformLink to="/performance/">performance monitoring feature</PlatformLink> turned on.

Reviewer: ```suggestion
On this page you'll learn how to manually propagate trace information into and out of your .NET application. Please note, that you don't need to do this manually, if you [use one of our supported frameworks](/platforms/dotnet/tracing/#how-to-use-tracing), or have our <PlatformLink to="/performance/">performance monitoring feature</PlatformLink> turned on.
```
